### PR TITLE
Try cloudevents-go with kn

### PR DESCRIPTION
**Pull request by mattmoor**
/hold

### CODE REVIEW

**Comment by mattmoor** [comment]
With a local render of the docs:

image

cc @rhuss @maximilien @abrennan89

**Comment by mattmoor** [comment]
I'm going to try staging the helloworld-go sample next.  Let's figure out the shape we want on these, then can "go broad" with the other languages in each of the respective categories.  Then we can go deep on the more focused samples.

WDYT?

**Comment by matzew** [comment]
I think this does look great, @mattmoor

**Comment by rhuss** [comment]
@mattmoor that looks nice, I think this is the way to go offer different alternatives for the same thing. Of course the next question will be what the default tab will be ;-) but this can be discussed on the hello world example.

**Comment by mattmoor** [comment]
I went with `kn` here, and (personally) I'd be happy to bias towards that, but if it'll be at all contentious, let's punt it for a follow-up.

**Comment by slinkydeveloper** [comment]
Cool! Can you do the same with other cloudevents-* examples?  
/lgtm

**Comment by mattmoor** [comment]
@slinkydeveloper I wanted to iterate on a single example of this and `helloworld-go` (stacked PR) as a template and then I was going to put together a spreadsheet to try and crowd source this to scale it horizontally.

Some thing will vary (e.g. `ko` here) and I'm not sure I'm the best to add that nuance in every language, but it should be easier to knock out than the original bootstrap.

File: docs/serving/samples/cloudevents/cloudevents-go/index.md [hunk 1]
```diff
@@ -5,4 +5,159 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
+A simple web app written in Go that can receive and send Cloud Events that you
```
Comment (n3wscott) [inline docs/serving/samples/cloudevents/cloudevents-go/index.md:8]: CloudEvents

**Comment by n3wscott** [comment]
one little nit, but /lgtm

**Comment by slinkydeveloper** [comment]
@mattmoor sgtm

File: docs/serving/samples/cloudevents/cloudevents-go/index.md (hunk 1 above)
Comment (mattmoor) [reply docs/serving/samples/cloudevents/cloudevents-go/index.md:8]: That's not really this PR, so you should check all the languages

Comment (n3wscott) [reply docs/serving/samples/cloudevents/cloudevents-go/index.md:8]: ok lgtm

**Comment by n3wscott** [comment]
/unhold

let it in, looks good

**Pull request merged**